buildSrc/src/main/java/Dependencies.kt

```diff
@@ -33,13 +33,13 @@ object Versions {
     const val androidx_work = "2.0.1"
     const val google_material = "1.1.0-alpha07"
 
-    const val mozilla_android_components = "6.0.2"
+    const val mozilla_android_components = "7.0.0-SNAPSHOT"
     // Note that android-components also depends on application-services,
     // and in fact is our main source of appservices-related functionality.
     // The version number below tracks the application-services version
     // that we depend on directly for tests, and it's important that it
     // be kept in sync with the version used by android-components above.
-    const val mozilla_appservices = "0.34.0"
+    const val mozilla_appservices = "0.36.0"
```

Comment:
Can you update the comment just above? we now also depend on it for the megazords.

Reply:
It still seems accurate to me, what would you prefer it to say?